doc-content/shared-kie-docs/src/main/asciidoc/KIE/BuildDeployUtilizeAndRun/kie-scanner-proc.adoc

```diff
@@ -6,7 +6,7 @@ The KIE scanner in {PRODUCT} monitors your Maven repository for new `SNAPSHOT` v
 
 [IMPORTANT]
 ====
-For production environments, do not use a KIE scanner or `SNAPSHOT` project versions to avoid accidental or unexpected project updates. The KIE scanner is intended for development environments that use `SNAPSHOT` project versions.
+For production environments, do not use a KIE scanner with `SNAPSHOT` project versions to avoid accidental or unexpected project updates. The KIE scanner is intended for development environments that use `SNAPSHOT` project versions.
```

Comment:
I guess that's fine, but it says something a little different. It makes it seem like you can use KIE scanner with production but just not with SNAPSHOT, but KIE Scanner is intended for use only with SNAPSHOT projects, which shouldn't ever be used in production. So really shouldn't be used in production at all, which is why I said "do not use a KIE scanner or SNAPSHOT projects" (neither of them) in production. If that's confusing though, then what you've changed is fine I suppose.

**Eder**, wdyt?

Reply:
I just sync with Pere and he thinks that is a good chance. So +1 for it.

Reply:
@sterobin 
> KIE Scanner is intended for use only with SNAPSHOT projects

Actually that is not truth. Kie scanner can be used also for release versions. See Automatic Update chapter in https://docs.jboss.org/jbpm/release/latest/jbpm-docs/html_single/#_updating_a_container